### ISSUE TITLE
Please add MyBookie

### ISSUE THREAD

**Issue by bigonsports** [acceptbitcoincash/acceptbitcoincash#285]
*Please fill in the template below, and don't worry about exact formatting, just do your best!

Requesting to add MyBookie.ag to the sports betting category. Details follow:

```yml

- name: MyBookie.ag
  url: http://record.webpartners.co/_wS-vFubHQx4WqcfzuvZcQGNd7ZgqdRLk/1/
  twitter: betmybookie
  facebook: betmybookie
  img: http://record.webpartners.co/_wS-vFubHQx4WqcfzuvZcQGNd7ZgqdRLk/1/
  bcc: yes
  btc: yes
  othercrypto: yes_or_no
  doc: https://mybookie.ag/info/banking/
```

Add more information here if necessary

**Comment by bigonsports** [comment]
Hi please add MyBookie

**Comment by bigonsports** [comment]
Thanks

**Comment by EchterAgo** [comment]
This is an affiliate link, it adds "?referredby=MYB100,_url" to the URL. I think the site should be linked directly. Also the site seems to restrict access from Thailand, so I can not see what it actually does.

**Comment by bigonsports** [comment]
Hi Axel,

Tracking link is needed to track number of clicks from 
acceptbitcoincash.com for marketing purposes. Please keep.

Not sure why you are unable to view from Thailand.

Thanks

John

**Comment by EchterAgo** [comment]
Why don't you just do referrer tracking like Google Analytics or any decent analytics package does? Anyway, if it is not a monetized referral link I don't see a problem with it.

As for why I can not access your website, I get this:

![image](https://user-images.githubusercontent.com/922233/33220343-0eb126f8-d17a-11e7-8757-37a01205ca5f.png)

Looks like Wordfence is blocking me for some reason.

**Comment by EchterAgo** [comment]
So, I think I know what is going on with the access problems. My internet providers IP ranges (I'm in 96.30.64.0 - 96.30.79.255) seem to be listed in quite a few spam blacklists. If you are using DNSBL in Wordfence to block people I would recommend against it. Better to ask people from such ranges to solve a captcha instead of just blocking them.

**Comment by bigonsports** [comment]
thanks, good to know.

**Comment by t0p0lin0** [comment]
MyBookie has been merged in #291
